src/Microsoft.Framework.PackageManager/NuGet/Core/Configuration/ConfigurationDefaults.cs

```diff
@@ -21,12 +21,9 @@ public class ConfigurationDefaults
 
         private static ConfigurationDefaults InitializeInstance()
         {
-#if DNX451
-            var commonAppData = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData);
-#else
-            var commonAppData = Environment.GetEnvironmentVariable("ProgramData");
-#endif
-            var baseDirectory = Path.Combine(commonAppData, "NuGet");
+            var machineWideSettingsDir = DnuEnvironment.GetFolderPath(DnuFolderPath.MachineWideSettingsDirectory);
+
+            var baseDirectory = Path.Combine(machineWideSettingsDir, "NuGet");
```

Comment:
It matters on case-sensitive filesystem. I filed a separate bug for it here: https://github.com/aspnet/dnx/issues/2270